docs/server/push-duty.rst

```diff
@@ -126,8 +127,18 @@ And the second::
     $ git tag 2015.09.10-2
     $ git push upstream 2015.09.10-2
 
-Then update the etherpad with the new comparison link for the updated tag.
+Then update the push doc with the new comparison link for the updated tag.
 
+Extension Workshop
+__________________
+
+The tag naming scheme for extension-workshop is a bit different. When tagging for
+stage, add the suffix ``-stage`` to the tag name. For example::
+
+    $ git checkout master
+    $ git pull
+    $ git tag 2022.02.10-stage
+    $ git push upstream 2022.02.10-stage
```

Comment:
I feel we probably want a clearer warning here that creating the tag without `-stage`, the normal thing for all the other repos, just deploys straight to prod with extension workshop.

Reply:
That is mentioned under `Push`, but that's a good idea to call it out here too. I'll add that.